src/optimum/intel/openvino/__init__.py

```diff
@@ -6,10 +6,13 @@
     OVAutoModelForSequenceClassification,
 )
 
+from .qa_quantize import OVQuantizerForQuestionAnswering as OVQuantizerForQuestionAnswering
```

Comment:
maybe just `from .qa_quantize import OVQuantizerForQuestionAnswering`?

Reply:
Thanks, fixed it